src/main/java/slimeknights/tconstruct/tables/recipe/TinkerStationPartSwapping.java

```diff
@@ -173,7 +174,8 @@ public RecipeResult<ItemStack> getValidatedResult(ITinkerStationContainer inv) {
           }
         }
         // everything worked, so good to go
-        return RecipeResult.success(tool.createStack(Math.min(inv.getTinkerableSize(), shrinkToolSlotBy())));
+        LazyToolStack lazyToolStack = LazyToolStack.from(tool, Math.min(inv.getTinkerableSize(), shrinkToolSlotBy()));
+        return RecipeResult.success(lazyToolStack);
```

Comment:
No need to split this over two lines, `LazyToolStack.from(tool, ` is not that much longer than `tool.createStack(`